lib/semantics/resolve-labels.cc

```diff
@@ -199,6 +199,8 @@ bool firstNameNoneOrBothEqual(
   }
 }
 
+unsigned sayLabel(parser::Label label) { return static_cast<unsigned>(label); }
```

Comment:
Prefer `int` to `unsigned` apart from low-level bitwise code.

Name functions with capital letters.

This function should be `static`.

Reply:
https://google.github.io/styleguide/cppguide.html#Integer_Types

Reply:
Labels are already unsigned. This function is simply to force a type agreement between the C printf() format specifier and the argument.

Reply:
Yes, will do.